src/net.h

```diff
@@ -344,7 +351,7 @@ class CNode
     const std::unique_ptr<TransportDeserializer> m_deserializer; // Used only by SocketHandler thread
     const std::unique_ptr<const TransportSerializer> m_serializer;
 
-    NetPermissionFlags m_permissionFlags{NetPermissionFlags::None}; // treated as const outside of fuzz tester
+    const NetPermissionFlags m_permissionFlags;
```

Comment:
nit (feel free to ignore), but it always felt weird to me to mix camels with snakes. Suggested commit:


```
    scripted-diff: Rename m_permissionFlags to m_permission_flags
    
    Seems better to use consistent style within one symbol name.
    Also, rename permissionFlags to permission_flags.
    
    -BEGIN VERIFY SCRIPT-
     sed -i 's/permissionFlags/permission_flags/g' $(git grep -l permissionFlags)
    -END VERIFY SCRIPT-

Reply:
Camels eaten by snakes.